### PR TITLE
add soft assert for subscription debugging

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?236652#1232177

@nickpell buddy @czue

### CODE REVIEW

File: corehq/apps/domain/models.py [hunk 1]
```diff
@@ -632,6 +632,10 @@ def save(self, **params):
         if not self._rev:
             # mark any new domain as timezone migration complete
             set_migration_complete(self.name)
+            # for http://manage.dimagi.com/default.asp?236652#1232177
+            # used for debugging why some domains are created without subs
+            _assert = soft_assert(notify_admins=True, exponential_backoff=False)
+            _assert(False, '%r was created' % name)
```
Comment (dimagimon) [inline corehq/apps/domain/models.py]: F821 undefined name 'name'

File: corehq/apps/style/templates/style/includes/global_navigation_bar.html [hunk 2]
```diff
@@ -56,6 +56,15 @@
                     <i class="fa fa-dashboard icon-dashboard dropdown-icon"></i> {% trans "Current Subscription" %}
                 </a>
             </li>
+            {% if request.couch_user_is_super_user %}
```
Comment (nickpell) [inline corehq/apps/style/templates/style/includes/global_navigation_bar.html]: is ```couch_user_is_super_user``` the intended syntax or did you mean for a period?

Comment (emord) [inline corehq/apps/style/templates/style/includes/global_navigation_bar.html]: removed it https://github.com/dimagi/commcare-hq/pull/13564/commits/4678473273844271b96dc45ff5d97125c1a142c5

**Review by nickpell — APPROVED** [review]

**Review by czue — APPROVED** [review]
any idea how long we are planning to leave this in? seems like this will be order of magnitude 30-40 emails per day (probably fine, just noting)

**Comment by emord** [comment]
@czue hopefully not long. it looks like these errors aren't uncommon, so it may pop up soon

**Comment by czue** [comment]
tests are failing left and right. i'm guessing you are going to have to add an unreasonable amount of `@softer_assert` decorators (maybe you can find a more clever workaround?)

**Comment by emord** [comment]
I'm not excited about using `UNIT_TESTING` in real code, but it does seem like it's done other places

**Comment by emord** [comment]
:white_check_mark:

**Pull request merged**